plugin/src/main/java/net/aufdemrand/denizen/events/player/PlayerKickedScriptEvent.java

```diff
@@ -63,28 +75,38 @@ public String getName() {
     public boolean applyDetermination(ScriptContainer container, String determination) {
         String lower = CoreUtilities.toLowerCase(determination);
         if (lower.startsWith("message:")) {
-            message = new Element(lower.substring(8));
+            message = new Element(lower.substring("message:".length()));
             return true;
         }
         else if (lower.startsWith("reason:")) {
-            reason = new Element(lower.substring(7));
+            reason = new Element(lower.substring("reason:".length()));
             return true;
         }
+        else if (lower.startsWith("fly_cooldown:")) {
+            Duration duration = Duration.valueOf(lower.substring("fly_cooldown:".length()));
+            if (duration != null) {
+                NMSHandler.getInstance().getPlayerHelper().setFlyKickCooldown(player.getPlayerEntity(), (int) duration.getTicks());
+                cancelled = true;
+                return true;
+            }
+        }
         return super.applyDetermination(container, determination);
     }
 
     @Override
     public ScriptEntryData getScriptEntryData() {
-        return new BukkitScriptEntryData(new dPlayer(event.getPlayer()), null);
+        return new BukkitScriptEntryData(player, null);
     }
 
     @Override
     public dObject getContext(String name) {
-        if (name.equals("message")) {
-            return message;
-        }
-        else if (name.equals("reason")) {
-            return reason;
+        switch (name) {
+            case "message":
+                return message;
+            case "reason":
+                return reason;
+            case "flying":
+                return new Element(isFlying());

```

Comment:
Don't reformat code that appears in 100 places without justifying it and applying it to all relevant cases.

Reply:
This was just an automatic suggestion by IntelliJ. I don't see any harm in changing these as we go along